project/Publish.scala

```diff
@@ -19,5 +21,33 @@ object Publish {
     publishArtifact in Test := false
   )
 
-  val settings = bintraySettings ++ publishingSettings
+  val updateReadmeVersion = ReleaseStep(action = s ⇒ {
+    val contents = IO.read(file("README.md"))
+
+    val p = Project.extract(s)
+    import p.get
+
+    val libDependencyPatten = "(\"" + get(organization) + "\"\\s+%+\\s+\"" + get(name) + "\"\\s+%\\s+\")[\\w\\.-]+(\")"
```

Comment:
typo: `Patten` -> `Pattern`

the regex would also be slightly more readable with `"""` (less escaped `\` and `"`)
```scala
s"""("${get(organization)}"\s+%+\s+"${get(name)}"\s+%\s+")[\w\.-]+(")"""
```